execution-scheduler/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup
+
+setup(
+    name='cloudify-execution-scheduler',
+    version='5.2.0.dev1',
+    author='Cloudify',
+    author_email='cosmo-admin@cloudify.co',
+    packages=['execution_scheduler'],
```

Comment:
use find_packages instead (`from setuptools import find_packages`) so that we dont have to change this, when we add more subpackages